pkgs/by-name/lx/lxc/user-nic.diff

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/lxc/network.c b/src/lxc/network.c
+index 0a99d32..850e975 100644
+--- a/src/lxc/network.c
++++ b/src/lxc/network.c
+@@ -2940,7 +2940,7 @@ int lxc_find_gateway_addresses(struct lxc_handler *handler)
+ 
+ #ifdef IN_LIBLXC
+ 
+-#define LXC_USERNIC_PATH LIBEXECDIR "/lxc/lxc-user-nic"
++#define LXC_USERNIC_PATH "lxc-user-nic"
```

Comment:
This also introduces potential side effects. Can this not be read from the `$out/libexec` ?

Reply:
These needs to point to the `lxc-user-nic` with the SUID set, otherwise it won't work. the one in `$out/libexec` does not have the SUID and so it won't create the lxc bridge.

Would it be better to set it to `/run/wrappers/bin/lxc-user-nic`?

Reply:
Makes sense. Yes we should explicitly use the wrappers path.